plugins/client/graphql-kotlin-client-generator/src/main/kotlin/com/expediagroup/graphql/plugin/client/generator/GraphQLClientGenerator.kt

```diff
@@ -196,6 +202,16 @@ class GraphQLClientGenerator(
         val rootType = operationNames[operationDefinition.operation.name]
         return graphQLSchema.getType(rootType).get() as ObjectTypeDefinition
     }
+
+    private fun parseSchema(path: String): TypeDefinitionRegistry {
+        val schemaFile = File(path)
+        return if (schemaFile.isFile) {
+            SchemaParser().parse(schemaFile)
+        } else {
+            val schemaInputStream = this.javaClass.classLoader.getResourceAsStream(path) ?: throw RuntimeException("specified schema file=$path does not exist")
```

Comment:
lets extract this exception to explicit `SchemaUnavailableException` (or something like that) under `com.expediagroup.graphql.plugin.client.generator.exceptions`